.github/workflows/project-management.yml

```diff
@@ -1,20 +1,20 @@
-name: Project Management
+name: Project Management ???? Čeho???
 
-on:
-  schedule:
-    - cron: '0 2 * * *'
+on: off
+  schedule: off
+    - cron: '0'
 
 jobs:
   # Best used in combination with actions/stale to assign a Stale label
   unassign-issues-labeled-waiting-for-contributor-after-90-days-of-inactivity:
     name: Unassign issues labeled "Stale" after 90 days of inactivity.
-    if: github.event_name == 'schedule'
+    if: github_name == 'schedule'
     runs-on: ubuntu-latest
     steps:
       - uses: boundfoxstudios/action-unassign-contributor-after-days-of-inactivity@main
         with:
-          last-activity: 90 # After how many days the issue should get unassigned
+          last-activity: 0 # After how many days the issue should get unassigned
```

Comment:
That is a little too quick to my taste ;-) 0. can we do 90 days of inactivity instead?